<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetFormDialog</class>
 <widget class="QDialog" name="TargetFormDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>225</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>225</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>225</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="passInput">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>184</y>
     <width>225</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>225</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>mot de passe</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hostInput">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>108</y>
     <width>225</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>225</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameInput">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>146</y>
     <width>225</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>225</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>identifiant</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathInput">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>222</y>
     <width>225</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>225</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>dossier de destination</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>260</y>
     <width>225</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="portLayout">
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="portLabel">
      <property name="text">
       <string>nÂ° Port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="portInput">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>21</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>dossier de destination</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>21</y>
     <width>225</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="typeLayout">
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="typeChoice">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">text-align:center</string>
      </property>
      <item>
       <property name="text">
        <string>NORMAL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FTP</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="tagInput">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>67</y>
     <width>225</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>225</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>etiquette</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dirChoice">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>105</y>
     <width>225</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>225</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>225</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>chemin</string>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <zorder>passInput</zorder>
  <zorder>hostInput</zorder>
  <zorder>usernameInput</zorder>
  <zorder>pathInput</zorder>
  <zorder>buttonBox</zorder>
  <zorder>portInput</zorder>
  <zorder>portLabel</zorder>
 </widget>
 <tabstops>
  <tabstop>typeChoice</tabstop>
  <tabstop>tagInput</tabstop>
  <tabstop>dirChoice</tabstop>
  <tabstop>hostInput</tabstop>
  <tabstop>usernameInput</tabstop>
  <tabstop>passInput</tabstop>
  <tabstop>pathInput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
